lib/matplotlib/style/core.py

```diff
@@ -34,11 +35,35 @@
 STYLE_FILE_PATTERN = re.compile('([\S]+).%s$' % STYLE_EXTENSION)
 
 
+# A list of rcParams that should not be applied from styles
+STYLE_BLACKLIST = set([
+    'interactive', 'backend', 'backend.qt4', 'webagg.port',
+    'webagg.port_retries', 'webagg.open_in_browser', 'backend_fallback',
+    'toolbar', 'timezone', 'datapath', 'figure.max_open_warning',
+    'savefig.directory', 'tk.window_focus', 'hardcopy.docstring'])
```

Comment:
In that case, I thought using style might be handy, e.g. an Emacs key style (not that one could do that given our current framework, but you get the idea).